external-crates/move/crates/move-analyzer/trace-adapter/src/utils.ts

```diff
@@ -20,3 +20,8 @@ export const FRAME_LIFETIME = -1;
  * The extension for JSON files.
  */
 export const JSON_FILE_EXT = ".json";
+
+/**
+ * The extension for trace files.
+ */
+export const TRACE_FILE_EXT = ".zst";
```

Comment:
Nit. It's a little inconsistent with Rust (and possibly a bit misleading) as it's not really the trace file extension but rather compressed file extension. Perhaps we could rename it accordingly?